sitesupport/common.js

```diff
@@ -53,16 +53,20 @@ function urlsafeB64Decode(data) {
   return atob(data.replace(/-/g, '+').replace(/_/g, '/'));
 }
 
-function basicBackendRequest(method, target, parameter, finish) {
+function basicBackendRequest(method, target, parameter, success, failure) {
   var xhttp = new XMLHttpRequest();
   xhttp.onreadystatechange = function() {
     if (this.readyState == 4 && this.status == 200) {
-      finish(this);
+      success(this);
       hideSpinner();
-    } else if (this.status == 404) {
-      alertbox('404!');
-    } else if (this.status == 409) {
-      alertbox('409!');
+    } else if (this.readyState == 4) {
```

Comment:
Out of curiosity, what exactly does `4` mean for `readyState`?

Perhaps, more broadly, we should introduce some constants here to make this self-documenting :-D